src/main/java/com/scalar/database/api/TransactionState.java

```diff
@@ -1,6 +1,6 @@
 package com.scalar.database.api;
 
-/** */
+/** The state of a transaction */
```

Comment:
I think it is better to describe in more detail because this state is used for two meanings, state of a transaction and state of a record.

Reply:
Thank you for your comment. I made a very simple change to "Flag to represent the state of a record in a transaction." I am not sure if it is worth adding anything on how the flags are used as this would be very large I think. I simply removed mention of "state of a transaction" as as far as I can tell it only represents the state of records.

Reply:
I often check this state if a record is consistent as expected.
I think a user wants to know what these states mean when the user checks the coordinator table or user tables.
How about adding descriptions for each state like [Consistency](https://github.com/scalar-labs/scalardb/blob/a4e223766cd703fabf0c95d82b547988d0760550/src/main/java/com/scalar/database/api/Consistency.java#L8)?

For example;

PREPARED state is for only a record. PREPARED state of a record shows that the record isn't committed because the corresponding transaction is in progress or has failed in the commit phase.